Week1/homework/createAndSeed.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+// REQUIRING SECTION
+const util = require('util');
+const { DATABASE_WORLD, COUNTRIES_TABLE, CITIES_TABLE } = require('./databaseAndTables');
+const { countries, cities } = require('./world_database');
+const connection = require('./connection');
+
+const execQuery = util.promisify(connection.query.bind(connection));
+
+async function seedDatabase() {
+  connection.connect();
+  try {
+    await execQuery(DATABASE_WORLD);
+    await execQuery(COUNTRIES_TABLE);
+    await execQuery(CITIES_TABLE);
+
+    countries.forEach(async country => {
+      await execQuery(`INSERT INTO countries SET ?`, country);
+      if (countries.indexOf(country) === countries.length - 1)
```

Comment:
Don't `indexOf` always give you 0 or 1? What do you want to compare here?

Reply:
Actually I've put this condition in order to console log a text once the last country of the countries array is been inserted.
I didn't intend to have a comparison here.